src/storage/redis_metadata.h

```diff
@@ -91,9 +93,9 @@ enum RedisCommand {
   kRedisCmdLMove,
 };
 
-const std::vector<std::string> RedisTypeNames = {"none",   "string",    "hash",      "list",
-                                                 "set",    "zset",      "bitmap",    "sortedint",
-                                                 "stream", "MBbloom--", "ReJSON-RL", "hyperloglog"};
+const std::vector<std::string> RedisTypeNames = {"none",      "string",      "hash",      "list",   "set",
+                                                 "zset",      "bitmap",      "sortedint", "stream", "MBbloom--",
+                                                 "ReJSON-RL", "hyperloglog", "tdigest"};
```

Comment:
```suggestion
                                                 "ReJSON-RL", "hyperloglog", "TDIS-TYPE"};
```

Reply:
Hi @PragmaTwice ,

Thanks for your suggestion! 😊

I have updated the type name of tdigest.

Best Regards,
Edward